_overviews/FAQ/index.md

```diff
@@ -277,3 +277,31 @@ give the compiler, then it's a compiler bug. Please report it on the
 [Scala 2 bug tracker](https://github.com/scala/bug/issues) or [Scala 3
 bug tracker](https://github.com/lampepfl/dotty/issues), but check
 first if it's a duplicate of an existing ticket.
+
+### I set a setting in sbt but nothing happened. Why?
+
+There could be a lot of reasons.  An extremely common one, that
+almost everyone runs into sooner or later, is that you have a bare
+setting in a multi-project build.
+
+For example, if you add this to your `build.sbt`:
+
+    scalaVersion := "2.13.7"
```

Comment:
For `scalaVersion` in particular, could we suggest

```scala
ThisBuild / scalaVersion := "2.13.7"
```

up front? If that fixes 80% of the users, they don't really need to read further explanation.

Also this is an own-goal problem created by hello-world.g8
https://github.com/scala/hello-world.g8/blob/e65d2a3d300af1c32bbd34037614daf62cd58686/src/main/g8/build.sbt#L4

Reply:
pushed an edit. wdyt?

Reply:
looks good.